src/bors/command/parser.rs

```diff
@@ -262,14 +280,26 @@ line two
 "#,
         );
         assert_eq!(cmds.len(), 1);
-        assert!(matches!(cmds[0], Ok(BorsCommand::Try { parent: None })));
+        assert!(matches!(
+            cmds[0],
+            Ok(BorsCommand::Try {
+                parent: None,
+                jobs: None
+            })
+        ));
     }
 
     #[test]
     fn parse_try() {
         let cmds = parse_commands("@bors try");
         assert_eq!(cmds.len(), 1);
-        assert!(matches!(cmds[0], Ok(BorsCommand::Try { parent: None })));
+        assert!(matches!(
```

Comment:
Rewriting these tests to `insta` would make it to bless them :laughing: (we don't have to do that in this PR).